src/index.js

```diff
@@ -0,0 +1,14 @@
+import _ from 'lodash';
+import './style.css';
+
+function component() {
+  const element = document.createElement('div');
+
+  // Lodash, now imported by this script
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+  element.classList.add('hello');
+
+  return element;
+}
```

Comment:
Refactor this into ES6 arrow function, that is 
` let component = () => {
       code goes here...
  }`

Reply:
Done